cmd/oras/internal/display/status/tty.go

```diff
@@ -143,3 +143,62 @@ func (ph *TTYPullHandler) TrackTarget(gt oras.GraphTarget) (oras.GraphTarget, St
 	ph.tracked = tracked
 	return tracked, tracked.Close, nil
 }
+
+// TTYCopyHandler handles tty status output for copy events.
+type TTYCopyHandler struct {
+	tty       *os.File
+	committed *sync.Map
```

Comment:
nit: we can use `sync.Map` directly.
```suggestion
	committed sync.Map
```

Reply:
Done